Assets/Scripts/NpcBehaviors/GoopNpc.cs

```diff
@@ -12,6 +12,12 @@ public class GoopNpc : BaseNpc
 {
     [SerializeField] private InventoryItemData _targetBookItem;
     private bool _hasBook;
+
+    protected override void Initialize()
```

Comment:
Same here, this function doesn't need to be overridden if it isn't doing anything unique